### DISCUSSION TITLE
moving mice between rigs?

### DISCUSSION THREAD

**Discussion by kimiline** [int-brain-lab/iblrig#579] [Q&A] [answered]
If we want to move mice between rigs using IBLRIGv8, is there anything special we need to do (i.e. in version 7 we copied the previous days session data to the new rig)?

**Comment by bimac** [comment, marked as answer]
Hi @kimiline. Sorry for the late reply - I missed the notification and it got lost over the holidays. 

As long as the data has been transferred to the local server (cf. https://int-brain-lab.github.io/iblrig/usage.html#copy-commands) you can resume training from any of your rigs. You don't need to -- and shouldn't -- copy any data manually. The copy scripts run several checks assuring consistency of your data-transfers.

I hope that answers your question. Cheers!
